TeslaLib/FileBaseTeslaOAuthTokenStore.cs

```diff
@@ -0,0 +1,214 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Threading.Tasks;
+using TeslaLib.Models;
+
+namespace TeslaLib
+{
+    public class FileBaseTeslaOAuthTokenStore : IOAuthTokenDataBase
```

Comment:
We already have a class named FileBasedOAuthTokenStore.  This name is too close and confusing.  Could you consider another name, like FileBasedTeslaOAuthTokenDataBase?

Reply:
Got it thanks, Done!